api_v3/lib/KalturaAssetService.php

```diff
@@ -119,8 +119,8 @@ protected function exportAction ( $assetId , $storageProfileId )
 	    {
 	        throw new KalturaAPIException(KalturaErrors::INVALID_FLAVOR_ASSET_ID, $assetId);
 	    }
-	    
-	    $dbStorageProfile = StorageProfilePeer::retrieveByPK($storageProfileId);	    
+		
+		$dbStorageProfile = StorageProfilePeer::retrieveByIdAndPartnerId($storageProfileId, $dbAsset->getPartnerId());
```

Comment:
Need to make sure that the partner ID of the KS is the asset owner, otherwise we still have an issue